static/src/js/components/Map/__tests__/ShapefileLayerExtended.test.js

```diff
@@ -0,0 +1,119 @@
+import { ShapefileLayerExtended } from '../ShapefileLayerExtended'
+
+describe('ShapefileLayerExtended', () => {
+  describe('separateMultiPolygons', () => {
+    test('should separate MultiPolygon features into individual Polygon features', () => {
+      const layer = new ShapefileLayerExtended({})
+      const geojson = {
+        type: 'FeatureCollection',
+        features: [
+          {
+            type: 'Feature',
+            properties: { name: 'Region Name' },
+            geometry: {
+              type: 'MultiPolygon',
+              coordinates: [
+                [[[0, 0], [1, 1], [1, 0], [0, 0]]],
+                [[[2, 2], [3, 3], [3, 2], [2, 2]]]
+              ]
+            }
+          }
+        ]
+      }
+
+      layer.separateMultiPolygons(geojson)
+
+      expect(geojson.features.length).toBe(2)
+      expect(geojson.features[0].geometry.type).toBe('Polygon')
+      expect(geojson.features[0].geometry.coordinates).toEqual([[[0, 0], [1, 1], [1, 0], [0, 0]]])
```

Comment:
is this supposed to be [0, 0], [1, 1], [1, 0], [0, 1]

Reply:
No, the last point just needs to be the same as the first. Then the middle points should just be distinct and go in a clockwise direction to be valid geojson. Although for the purposes of the test, it wouldn't matter.